configuration/uart.md

```diff
@@ -1,99 +1,100 @@
 # UART configuration
 
-There are two types of UART available on the Raspberry Pi -  [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html) and mini UART. The PL011 is a capable, broadly 16550-compatible UART, while the mini UART has a reduced feature set.
+A [UART](https://en.wikipedia.org/wiki/Universal_asynchronous_receiver-transmitter) is a serial communication device which uses one or more signal wires and a common ground to transmit and receive data between two devices. When computer people talk about 'serial ports', they usually mean UARTs. People from DOS/Windows background may know these as 'COM ports'.
 
-All UARTs on the Raspberry Pi are 3.3V only - damage will occur if they are connected to 5V systems. An adaptor can be used to connect to 5V systems. Alternatively, low-cost USB to 3.3V serial adaptors are available from various third parties.
+## Hardware
 
-## Pi Zero, 1, 2 and 3 - two UARTs
+On all Raspberry Pi computers except microcontrollers, there are at least two UARTs built into the [SoC](https://en.wikipedia.org/wiki/System_on_a_chip). The Raspberry Pi 4 family has six UARTs.
 
-The Raspberry Pi Zero, 1, 2, and 3 each contain two UARTs as follows:
+There are also two different types of UART devices available on the Raspberry Pi:
+* [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html) part of the ARM spec and is patterned on the full-featured [16550 UART](https://en.wikipedia.org/wiki/16550_UART) chip.
```

Comment:
To be pedantic: not "part of the ARM spec", just a peripheral block available from ARM which plugs in nicely to the rest of the AMBA ecosystem.

Reply:
Thanks, will be changed in next rev.